Form/Type/VichImageType.php

```diff
@@ -61,9 +61,9 @@ public function buildView(FormView $view, FormInterface $form, array $options):
                     throw new \RuntimeException('LiipImagineBundle must be installed and configured for using "imagine_pattern" option.');
                 }
 
-                $uri = $this->storage->resolveUri($object, $form->getName());
-                if ($uri) {
-                    $view->vars['image_uri'] = $this->cacheManager->getBrowserPath($uri, $options['imagine_pattern']);
+                $path = $this->storage->resolvePath($object, $form->getName(), null, true);
+                if ($path) {
```

Comment:
Can we avoid implicit casting here? `null !== $path` would be better

Reply:
Done